docs/dashboards/custom-component.md

```diff
@@ -52,11 +52,9 @@ class YouTubeComponent extends Component {
 
     resize(width, height) {
         super.resize.call(this, width, height);
-        this.youTubeElement.setAttribute('width', width - 10);
-        this.youTubeElement.setAttribute('height', height - 10);
     }
```

Comment:
Resize method is not needed to be overriden if it calls only super with the same arguments.